tests/Env/DumperTest.php

```diff
@@ -86,7 +86,7 @@ private function createEnv()
             ->willReturn(null);
 
         $env = EnvFactory::createEnv(
-            EnvEnum::create(EnvEnum::SYMFONY),
+            EnvName::SYMFONY(),
```

Comment:
I'd recommend to add the methods in the enum types docblock if we use this notation, as described in https://github.com/Elao/PhpEnums#shortcuts to keep static code analysis and autocompletion viable :)